commit be8201e8caed41df3f646bd2be8bb9823c26711d (refs/changes/31/1431/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-02-03T14:32:35-08:00 (2 years, 8 months ago)
    
    IMGAPI-613 the 'import_image' task of the import-remote-image workflow has too short of a timeout

diff --git a/lib/workflows/import-remote-image.js b/lib/workflows/import-remote-image.js
index 150a6aa..c4291db 100644
--- a/lib/workflows/import-remote-image.js
+++ b/lib/workflows/import-remote-image.js
@@ -196,7 +196,7 @@ var workflow = module.exports = {
         }
     }, {
         name: 'import_image',
-        timeout: 10,
+        timeout: 60,
         retry: 1,
         body: importImage,
         modules: { sdcClients: 'sdc-clients' }
@@ -216,13 +216,13 @@ var workflow = module.exports = {
     timeout: 7250,
     onerror: [ {
         name: 'delete_image',
-        timeout: 10,
+        timeout: 60,
         retry: 1,
         body: deleteImage,
         modules: { sdcClients: 'sdc-clients' }
     }, {
         name: 'origin_delete_images',
-        timeout: 10,
+        timeout: 60,
         retry: 1,
         body: originDeleteImages,
         modules: {
